Tasks/AzureRmWebAppDeployment/operations/TaskParameters.ts

```diff
@@ -54,11 +60,36 @@ export class TaskParametersUtility {
 
         return taskParameters;
     }
+
+    private static _initializeDefaultParametersForPublishProfile(taskParameters: TaskParameters): void {
+        taskParameters.PublishProfilePath = tl.getInput('PublishProfilePath', true);
+        taskParameters.PublishProfilePassword = tl.getInput('PublishProfilePassword', true);
+        taskParameters.isLinuxApp = false;
+        taskParameters.isBuiltinLinuxWebApp = false;
```

Comment:
I dont understand why do you need to initialize rest of the inputs. Only 1st two or that are required for the scenario - should be sufficient

Reply:
done. But I feel it is good to initialize variables so it doesn't create any unexpected behavior in the future while making code changes.